packages/salesforcedx-sobjects-faux-generator/scripts/regenerateMinsSObjects.ts

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/**
+ * Simple script to re-generate file `minSObjects.json`
+ *
+ * To run:
+ *    npx ts-node ./scripts/regenerateMinsSObjects.ts <token> <instance_url>
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import { SObjectShortDescription } from '../src/describe';
+import { OrgObjectDetailRetriever } from '../src/retriever';
+import { SObject } from '../src/types';
+
+import { AuthInfo, Connection } from '@salesforce/core';
+
+// tslint:disable-next-line:no-floating-promises
+(async () => {
+  const args = process.argv.slice(2);
+
+  if (args.length <= 1) {
+    console.log(
+      'Usage:\n ./scripts/' +
+        path.basename(__filename) +
+        ' <auth_token> <instance_url>'
+    );
+    process.exit(1);
+  }
+  const token = args[0];
+  const instanceUrl = args[1];
+
+  const connection = await createConnection(token, instanceUrl);
+
+  await generateLocalSobjectJSON(connection);
+})();
+
+async function createConnection(token: string, instanceUrl: string) {
+  return await Connection.create({
```

Comment:
I prefer to use a pattern like:
    **const connection = (await Org.create({ aliasOrUsername })).getConnection();**

It's simpler, and you don't need to pass around a token you can just use a connection alias that you establish in your environment.

Reply:
👍  much better now. Thanks!